packages/storybook/stories/toast/examples.stories.tsx

```diff
@@ -124,6 +124,9 @@ export function WithCourierProvider({ bodyText }) {
               message:
                 "https://d33wubrfki0l68.cloudfront.net/ca2747f11cc64d0e424e27b4a804b9d981b22453/9ab46/_next/static/images/logo@2x-5d5af82635bfdd3ad24e54f9eb364097.png",
             },
+            toast: {
+              autoCloseTimer: 0,
```

Comment:
I was not able to see inbox examples in deploy preview for some reason?

Reply:
its because the preview is missing env variables so the story borks